src/views/clusteroverview/SettingsTab/ClusterTab/components/VirtualizationFeaturesSection/VirtualizationFeaturesWizard/components/ConfigurationStep/components/HighAvailabilityConfigurationSection/components/HighAvailabilityToggleContent/SwitchWithTooltip.tsx

```
@@ -18,12 +20,13 @@ const SwitchWithTooltip: FC<SwitchWithTooltipProps> = ({
   switchIsDisabled ? (
     <Tooltip content={disabledTooltipContent}>
       <Switch
+        data-test-id={dataTestID}
         isChecked={switchState}
         isDisabled={switchIsDisabled}
         onChange={(_, checked: boolean) => onSwitchChange(checked)}
       />
     </Tooltip>
   ) : (
-    <Switch isChecked={switchState} onChange={(_, checked: boolean) => onSwitchChange(checked)} />
+    <Switch data-test-id={dataTestID} isChecked={switchState} onChange={(_, checked: boolean) => onSwitchChange(checked)} />
```

ESLint

[error] Replace `·data-test-id={dataTestID}·isChecked={switchState}·onChange={(_,·checked:·boolean)·=>·onSwitchChange(checked)}` with `⏎······data-test-id={dataTestID}⏎······isChecked={switchState}⏎······onChange={(_,·checked:·boolean)·=>·onSwitchChange(checked)}⏎···`

(prettier/prettier)